Dockerfile.amd64

```diff
@@ -12,7 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1-407
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.2-301.1592810506 AS base
+
+COPY k8s-install/scripts/clean.sh /clean.sh
+
+RUN touch /in-the-container
```

Comment:
What's the purpose of these various RUN parms?  Is it just to check the executables can run?  Couldn't we do that in some container verification post build?